modules/openapi-generator/src/test/resources/2_0/swift/petstore-with-fake-endpoints-models-for-testing.yaml

```diff
@@ -1212,6 +1214,7 @@ definitions:
         readOnly: true
         type: integer
         format: int32
+        x-null-encodable: true
```

Comment:
@jarrodparkes it would be nice to have a property that is nullable with the default value of null with `x-null-encodable: true`, to make the test case broader

Reply:
fixed